docs/mkDocs/docs/options.md

```diff
@@ -733,3 +733,52 @@ an import of a module coming from package `PACKAGE`. e.g.
 `--exclude-automatic-assumptions-for=vector` would stop loading
 `_LHAssumptions` modules for any imports coming from package
 `vector`.
+
+## Warn on Term Holes
+
+**Options:** `warn-on-term-holes`
+
+LiquidHaskell can be configured to emit warnings whenever it encounters incomplete terms, known as *holes*.
+
+This flag is particularly useful during development, ensuring that any placeholders in your specifications are reviewed and completed before final evaluation.
+
+To activate this behavior, use the `--warn-on-term-holes` flag either:
+
+1. As a pragma in your source file:
+    ```haskell
+    {-@ LIQUID "--warn-on-term-holes" @-}
+    ```
+2. As a plugin option:
+    ```
+    ghc-options: -fplugin-opt=LiquidHaskell:--warn-on-term-holes
+    ```
+
+When enabled, LiquidHaskell issues a warning for each term hole it discovers, for example:
```

Comment:
Showing the source code here might be helpful to make sense of the error message that follows.